cmd/operator/main.go

```diff
@@ -88,12 +91,20 @@ func Main() int {
 		fmt.Fprint(os.Stderr, "`--configmap` flag is required, but not specified.")
 	}
 
+	r := prometheus.NewRegistry()
+	r.MustRegister(prometheus.NewGoCollector())
```

Comment:
I think we might have done the same mistake in the Prometheus Operator, but we should also add the process collector here. https://github.com/prometheus/client_golang/blob/bcbbc08eb2ddff3af83bbf11e7ec13b4fd730b6e/prometheus/registry.go#L55

Reply:
Sounds good :+1: I'll make sure to add this to Prometheus Operator as well.